### PR TITLE
Added booting documentation for Alibaba Cloud

### PR DESCRIPTION
**Pull request by robertvolkmann**
Fixes: https://github.com/coreos/fedora-coreos-docs/issues/115

### CODE REVIEW

File: modules/ROOT/pages/provisioning-aliyun.adoc [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
++
+.Example deleting uploaded blob
+[source, bash]
+----
+aliyun oss rm "${BUCKET_URL}/${IMAGE_BLOB}" --region "${REGION}"
+----
+
+== Creating a VSwitch (Optional)
+
+There exists no default VPCs or VSwitches in Alibaba Cloud. Hence for creating any instances a VSwitch has to exists. Pick some existing or create one with the following steps.
```
Comment (lucab) [inline modules/ROOT/pages/provisioning-aliyun.adoc:81]: I seem to remember instances can be created in a standalone way with a floating public IP directly attached to them, am I wrong?
While I acknowledge that a VPC is a better infrastructure, standalone addressable nodes are usually easier to grasp for people that are getting started. What about documenting that first, and leaving the VPC setup to a later advanced section?

(Assuming my memory is not mixing details from other clouds, in which case just disregard this comment)

**Comment by lucab** [comment]
Thanks a lot for this PR!
I don't have an AliCloud account at hand right now, but I'll get back here soon as I was planning to touch this anyway in the next weeks.

**Comment by dustymabe** [comment]
Thanks so much @robertvolkmann ! 🎉🎉🎉

File: modules/ROOT/pages/provisioning-aliyun.adoc (hunk 1 above)
Comment (robertvolkmann) [reply modules/ROOT/pages/provisioning-aliyun.adoc:81]: I don't know anything about something like "standalone addressable nodes". I never saw such an option in the Console or in the API documentation. Currently, only VPC networks are supported, see [https://www.alibabacloud.com/help/doc-detail/61651.htm](Network Types).

**Comment by robertvolkmann** [comment]
@lucab Did you get a chance to look into it?

**Comment by lucab** [comment]
@robertvolkmann not yet. The company account I used initially when working on AliCloud is gone. I asked for access to another internal one we have, and I hope to be unblocked next week.

**Review by lucab — APPROVED** [review]

**Pull request merged**